src/core/lib/security/security_connector/load_system_roots_linux.cc

```diff
@@ -20,26 +20,25 @@
 
 #include "src/core/lib/security/security_connector/load_system_roots_linux.h"
 
-#include <grpc/slice_buffer.h>
+#include <stdio.h>
```

Comment:
I think these should be inside the `#if defined(GPR_LINUX) || defined(GPR_ANDROID)` block.

Reply:
fixed